### PR TITLE
Add Twig support

### PR DESCRIPTION
**Pull request by RobLoach**
This just does locals, none of the more complex features that [Twig.js](https://github.com/justjohn/twig.js) provides.

cc @madeofpeople

### CODE REVIEW

**Comment by florianbouvot** [comment]
+1

**Comment by danicholls** [comment]
+1 pretty plz

**Comment by danicholls** [comment]
(Or actually use twig.js?)

File: lib/consolidate.js [hunk 1]
```diff
@@ -464,6 +464,31 @@ exports.liquor.render = function(str, options, fn){
 };
 
 /**
+ * Twig support,
+ */
+
+exports.twig = fromStringRenderer('twig');
+
+/**
+ * Liquor string support.
```
Comment (BridgeAR) [inline lib/consolidate.js]: Typo

**Comment by BridgeAR** [comment]
Would you be so kind and squash or fixup your commits? :)

**Comment by RobLoach** [comment]
Squashed.

**Comment by BridgeAR** [comment]
Thanks :+1:

**Pull request merged**